### PR TITLE
Update database models and add helper functions

### PR DESCRIPTION
**Pull request by markhulia**

### CODE REVIEW

File: hlpr/services/db/models.py [hunk 1]
```diff
@@ -1,31 +1,33 @@
...
 
 class Helper(User):
     __tablename__ = "helper"
     id = Column(Integer, primary_key=True)
-    tasks = relationship("Service")
-    rating = relationship("Rating")
+    avg_rating = Column(Float)
+    tasks = relationship("Task", backref="helper", lazy="select")
+    rating = relationship("Rating", backref="helper", lazy="select")
+    reviews = Column(String(512))
```
Comment (LeonardAukea) [inline hlpr/services/db/models.py:30]: What is this supposed to be? In the `user.proto` its an array of review objects. They would all have an UUID though

File: hlpr/services/db/models.py [hunk 2]
```diff
@@ -34,33 +36,43 @@ def __str__(self):
...
     def __str__(self):
-        return f"Task: {self.task}, created by {self.shopper} is picked up by " \
-               f"{self.helper}"
+        return (
+            f"Task: {self.task}, created by {self.shopper} is picked up by "
+            f"{self.helper}"
+        )
 
 
-class Rating:
```
Comment (LeonardAukea) [inline hlpr/services/db/models.py:57]: rename to review

File: hlpr/services/db/models.py [hunk 3]
```diff
@@ -1,31 +1,33 @@
...
+    phone_number = Column(String(24), index=True, unique=True)
     country = Column(String(64), index=True, unique=True)
     postCode = Column(Integer)
-    registered = Column(DateTime, default=datetime.datetime.utcnow)
+    joined = Column(DateTime, default=datetime.datetime.utcnow())
     lat = Column(Float)
     lon = Column(Float)
 
 
 class Helper(User):
```
Comment (LeonardAukea) [inline hlpr/services/db/models.py:24]: Why separate User into `Helper` `Shopper` I think `User` is enough

File: hlpr/services/db/models.py [hunk 4]
```diff
@@ -1,31 +1,33 @@
...
 from sqlalchemy import create_engine
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import relationship
 
-ENGINE = create_engine("sqlite:///my.db", echo=True)
+ENGINE = create_engine("sqlite:///helpr.db", echo=True)
 BASE = declarative_base()
 
 
 class User(BASE):
```
Comment (LeonardAukea) [inline hlpr/services/db/models.py:12]: Role is missing

File: hlpr/services/db/models.py (hunk 3 above)
Comment (markhulia) [reply hlpr/services/db/models.py:24]: Because they have different attributes (ratings for example)

File: hlpr/services/db/models.py (hunk 4 above)
Comment (markhulia) [reply hlpr/services/db/models.py:12]: Those are two separate models

File: hlpr/services/db/models.py (hunk 2 above)
Comment (markhulia) [reply hlpr/services/db/models.py:57]: ok

**Pull request merged**